pcsx2/gui/AppCoreThread.cpp

```diff
@@ -427,23 +427,29 @@ static void _ApplySettings( const Pcsx2Config& src, Pcsx2Config& fixup )
 		// the BIos isn't loaded until after initial calls to ApplySettings.
 
 		gameName = L"Booting PS2 BIOS... ";
-	}
 
 	//Till the end of this function, entry CRC will be 00000000
-	if (!gameCRC.Length())
+	CDVD_SourceType cdvdsrc(g_Conf->CdvdSource);
+	if ((!gameCRC.Length()) && ((cdvdsrc != CDVD_SourceType::NoDisc)))
 	{
 		Console.WriteLn(Color_Gray, "Patches: No CRC found, using 00000000 instead.");
 		gameCRC = L"00000000";
 	}
+		else if ((!gameCRC.Length()) && ((cdvdsrc == CDVD_SourceType::NoDisc)))
+		gameCRC = L"00000000";
+	}
 
 	// regular cheat patches
-	if (fixup.EnableCheats)
+	CDVD_SourceType cdvdsrc(g_Conf->CdvdSource);
+	if ((fixup.EnableCheats) && (cdvdsrc != CDVD_SourceType::NoDisc))
+	{
 		gameCheats.Printf(L" [%d Cheats]", LoadPatchesFromDir(gameCRC, GetCheatsFolder(), L"Cheats"));
+	}
+		else if ((fixup.EnableCheats) && (cdvdsrc == CDVD_SourceType::NoDisc))
```

Comment:
It'll either do nothing or block widescreen patches when cheats are enabled. Plus you need to look at the spacing of several lines, there's a tab too much at the beginning of this line for example(though the whole line should be removed).